src/main/java/bio/terra/service/SamClientService.java

```diff
@@ -245,6 +231,18 @@ public void createStudyResource(AuthenticatedUserRequest userReq, UUID studyId)
         ResourcesApi samResourceApi = samResourcesApi(userReq.getToken());
         logger.debug(req.toString());
         createResourceCorrectCall(samResourceApi.getApiClient(), ResourceType.STUDY.toString(), req);
+
+        // we'll want all of these roles to have read access to the underlying data, so we sync and return the emails
+        // for the policies that get created by SAM
+        ArrayList<String> rolePolicies = new ArrayList<>();
+        for (DataRepoRole role : Arrays.asList(DataRepoRole.STEWARD, DataRepoRole.CUSTODIAN, DataRepoRole.INGESTER)) {
+            Map<String, List<Object>> results = samGoogleApi(userReq.getToken()).syncPolicy(
+                ResourceType.STUDY.toString(),
+                studyId.toString(),
+                role.toString());
+            rolePolicies.add(results.keySet().iterator().next());
```

Comment:
Maybe explain what this is doing? Why is getting the `next` the right key to get? Are you guaranteed there is only one?  Some comments here would help.

Reply:
Yep, good point. This is copying another piece of code that is also not documented -- probably worth factoring out into a separate method with docs.